### PR TITLE
docs: Update Gateway pages for V3

### PR DESCRIPTION
**Pull request by derrekcoleman**
Updates the Gateway pages in the Learn and Build sections to reflect Gateway V3's new functionality.

The *dev* step-by-step guide has been updated for the SDK. I researched whether it's possible to add typesense to these codeblocks; it would be nice if readers can hover over the parameters and see their custom type traits. I wasn't able to find anything in the Docusaurus ecosystem.

The *user* step-by-step guide update can come later, once our app has the LSTs available.

TODO: DocGen based on TSDoc

### CODE REVIEW

File: docs/docs/build/bob-sdk/gateway.md [hunk 1]
```diff
@@ -2,164 +2,135 @@
...
-$ yarn add @gobob/bob-sdk
-
-# with npm
-$ npm i @gobob/bob-sdk
-
-# with pnpm
-$ pnpm add @gobob/bob-sdk
-
-# with Bun
-$ bun add @gobob/bob-sdk
```
Comment (gregdhill) [inline docs/docs/build/bob-sdk/gateway.md:40]: Do we wanna keep this?

File: docs/docs/build/bob-sdk/gateway.md [hunk 2]
```diff
@@ -2,164 +2,135 @@
...
-
-  const signedTx = Transaction.fromPSBT(psbt);
-
-  return Buffer.from(signedTx.extract())
-}
+```ts
+import { base64 } from "@scure/base";
+import { Transaction } from "@scure/btc-signer";
+
+// NOTE: It is up to your implementation to sign the PSBT here!
```
Comment (gregdhill) [inline docs/docs/build/bob-sdk/gateway.md:96]: I know it's not ready yet but we should really detail how to also integrate sats-wagmi as a default example to do this

File: docs/docs/learn/guides/bob-gateway/index.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
-It's built on a trustless, RFQ-based cross-chain [swap protocol](../../../build/examples/btc-swap/) that connects professional LPs with users through a seamless swapping experience. Essentially, LPs handle the complexities of bridging on behalf of users in exchange for a fee.
+It's built on a trustless, RFQ-based cross-chain [swap protocol](../../../build/examples/btc-swap/) that connects professional LPs with users through a seamless swapping experience. Essentially, LPs handle the complexities of bridging and staking on behalf of users in exchange for a fee.
 
 :::tip Interested in providing liquidity?
 If you are interested in being an LP for the BOB Gateway bridge, please send us an email at `gateway [ at ] gobob.xyz`.
 :::
 
-All you need is a Bitcoin wallet with some BTC to send and an EVM-compatible wallet to receive your wrapped Bitcoin on BOB. We'll even send you some ETH to cover the fees of your first few transactions on BOB.
+All you need is a Bitcoin wallet with some BTC to send and an EVM-compatible wallet to receive your Bitcoin LST/LRT on BOB. We'll even send you some ETH to cover the fees of your first few transactions on BOB.
```
Comment (gregdhill) [inline docs/docs/learn/guides/bob-gateway/index.md:18]: It's both right?

Comment (gregdhill) [reply docs/docs/learn/guides/bob-gateway/index.md:18]: wrapped and LST/LRT

File: docs/docs/build/bob-sdk/gateway.md (hunk 2 above)
Comment (derrekcoleman) [reply docs/docs/build/bob-sdk/gateway.md:96]: Agreed. Let's add the sats-wagmi example asap in another PR.

File: docs/docs/learn/guides/bob-gateway/index.md (hunk 3 above)
Comment (derrekcoleman) [reply docs/docs/learn/guides/bob-gateway/index.md:18]: Agreed; added that back in. Emphasis on the new feature shouldn't come at the cost of clarity of options.

File: docs/docs/build/bob-sdk/gateway.md (hunk 1 above)
Comment (derrekcoleman) [reply docs/docs/build/bob-sdk/gateway.md:40]: The `npm2yarn` package provides a tab-switcher between package managers, so we still have multiple options. I opted for that to strive for visual conciseness, but I'm happy to revert if you prefer.

**Review by gregdhill — APPROVED** [review]

**Pull request merged**